test/models/earnings_summary_test.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class EarningsSummaryTest < ActiveSupport::TestCase
+  test "returns empty array when there are no fee transactions" do
+    portfolio = create(:portfolio)
+    # Create transactions with different types (not fee)
+    create(:portfolio_transaction, :deposit, portfolio: portfolio)
+    create(:portfolio_transaction, :debit, portfolio: portfolio)
+
+    result = EarningsSummary.new(portfolio).earnings_breakdown
+    assert_equal [], result
+  end
+
+  test "returns earnings breakdown for single reason" do
+    portfolio = create(:portfolio)
+    create(:portfolio_transaction, portfolio: portfolio, transaction_type: :fee,
```

Comment:
https://github.com/rubyforgood/stocks-in-the-future/blob/8627649575987e8ff487bf83c6d39083009d0838/app/services/distribute_earnings.rb#L47-L51

The transaction type is `:deposit`, not `:fee`, so it's false positive test case